client/src/components/Guests.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  border-top: 0.4px solid rgba(0, 0, 0, 0.12);
+`;
```

Comment:
could put padding 10px in Container instead of both InfoContainer and ArrowContainer